dashboards/grafana-dashboard-assisted-resource-dashboard.yaml

```diff
@@ -2,6 +2,48 @@ apiVersion: v1
 data:
   assisted-installer.json: |-
     {
+      "__inputs": [
+        {
+          "name": "DS_APP-SRE-STAGE-01-PROMETHEUS",
```

Comment:
@yobshans can you explain this change?
it looks like it would work only with a hardcoded datasource.

I have uploaded this to a dev-grafana, and I see only the stage datasource is working.

Reply:
@gamli75 FYI

Reply:
Hey, probably this is a result when I export Dashboard with selected stage datasource.
Need take a look previous version and update header with correct input
        "datasource": "-- Grafana --",